component/lambda/functions/github_lambda_loader.py

```diff
@@ -0,0 +1,30 @@
+import importlib, urllib.request, urllib.parse, hashlib, pathlib, os, sys, tempfile, logging
+
+logger = logging.getLogger()
+logger.setLevel("INFO")
+
+# The name of the file containing the lambda handler (defaults to <lambda_function_name>.py)
+SI_IMPORT_LAMBDA_HANDLER = os.getenv('SI_IMPORT_LAMBDA_HANDLER') or f"{os.environ['AWS_LAMBDA_FUNCTION_NAME']}.py"
+# Where to download the Python files from
+SI_GITHUB_FUNCTIONS_URL = os.getenv('SI_GITHUB_FUNCTIONS_URL') or "https://raw.githubusercontent.com/systeminit/si/refs/heads/main/component/lambda/functions/"
```

Comment:
nitpick: I think I'd prefer these not to have a default and just bomb out if not set

Reply:
For SI_GITHUB_FUNCTIONS_URL I think that makes a lot of sense. For SI_IMPORT_LAMBDA_HANDLER I kind of liked enforcing a convention where you name the script the same as the lambda--I just added the env var in case someone ever wanted to do something weird. (Same with the list of libraries)